client/web/src/enterprise/batches/detail/changesets/EmptyDraftChangesetListElement.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { useLocation } from 'react-router'
+
+import { Link } from '@sourcegraph/wildcard'
+
+import styles from './EmptyDraftChangesetListElement.module.scss'
+
+export const EmptyDraftChangesetListElement: React.FunctionComponent<{}> = () => {
+    const location = useLocation()
+    return (
+        <div className={styles.emptyDraftChangesetListElementBody}>
+            <h3 className="mb-2">No changesets exist</h3>
+            <div className={styles.emptyDraftChangesetListElementContent}>
+                <span>This batch change is a draft. A batch specification must be executed to create changesets.</span>
+                <Link to={`${location.pathname}/edit`}>View the most recent specification.</Link>
```

Comment:
I know this is the copy from the Figma, but we keep telling Rob we don't actually call it a "batch specification" anywhere, we only call it a "batch spec." 😂 I know, I know, abbreviations are bad, but inconsistency is worse maybe? Anyway:

```suggestion
                <span>This batch change is a draft. A batch spec must be executed to create changesets.</span>
                <Link to={`${location.pathname}/edit`}>View the most recent spec.</Link>
```

Reply:
Thank you Kelli 😆 Now I am finally not the only one griping about this all the time :phew:

Reply:
Haha! Sure I'll update this.